### ISSUE TITLE
Why the 8 category limit?

### ISSUE THREAD

**Issue by man-netcat** [codestation/gclite#14]
I'm interested in this from a technical view, is it related to the XMB somehow?

**Comment by codestation** [comment]
Yes, i don't remember well but it crashed my psp slim when using more than 8 categories in folder mode. Maybe the vsh could be patched to increase the limit but you could use the regular listing mode instead.
